### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by whiteplainshistory** [community/community#21734] [Other Feature Feedback, Questions, & Ideas]
<p>I uploaded an identical copy of a file with a new name.  In particular, <a href="https://whiteplainshistory.github.io/exhibits/little/LittleFarm.html" rel="nofollow noopener">https://whiteplainshistory.github.io/exhibits/little/LittleFarm.html</a> displays while <a href="https://whiteplainshistory.github.io/exhibits/little/LittleFarm2.html" rel="nofollow noopener">https://whiteplainshistory.github.io/exhibits/little/LittleFarm2.html</a> gives 404.  It’s been over an hour.  What’s the deal?</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Did you check your email for a page build failure?  I had no problems with your pages (as LittleFarm or LittleFarm2) once I fixed my build error.</p>
<p>A more descriptive subject line besides ‘404’ might get you more attention.</p>

**Comment by whiteplainshistory** [comment]
<p>I see now, thanks for the pointer!</p>
